nuget/nunitlite.nuspec

```diff
@@ -8,7 +8,7 @@
     <owners>Charlie Poole</owners>
     <licenseUrl>http://nunit.org/nuget/nunit3-license.txt</licenseUrl>
     <projectUrl>http://nunit.org</projectUrl>
-    <iconUrl>http://nunit.org/nuget/nunit_32x32.png</iconUrl>
+    <iconUrl>http://nunit.org/nuget/nunitv3_32x32.png</iconUrl>
```

Comment:
It's odd that you show this and the other changes to the nuspec files. They are already changed in master. I'm guessing that this is because you didn't do a new PR. Because of how GitHub handles diffs in PRs, it's probably best in any situation like this, where the PR has gotten far behind, to keep the old branch but delete and re-create the PR.

That's for next time. The changes here are minimal so it's easy enough to review.